lib/ui/welcome/payable/payable_viewmodel.dart

```diff
@@ -35,11 +35,30 @@ class PayableViewModel extends BaseViewModel {
     return _payable;
   }
 
+  // initializing state to work with button on keypad
+  String _amount = '';
+  String get amount {
+    return _amount;
+  }
+
+  bool _showButton = false;
+  bool get showButton {
+    return _showButton;
+  }
+
   int _total;
   int get total {
     return _total;
   }
 
+//TODO:{Telesphore}  Work on Keypad --> get information from typed value
+//FIXME: {Telesphore} fix the responsiviness of keypad
```

Comment:
I like that we keep something to do documented so we can not forget. Good @tuganimana